tests/nightly/test_large_array.py

```diff
@@ -279,6 +279,19 @@ def test_diag():
     assert_almost_equal(r.asnumpy(), np.diag(a_np, k=k))
 
 
+def test_softmax():
+    def softmax_forward(input_data, true_output):
```

Comment:
If this is called once, do we need this sub routine?

Reply:
ok